assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -14528,7 +14528,9 @@ else if (m_criteria.equals(SORTED_GRADE_SUBMISSION_CONTENTREVIEW)) {
                         List<ContentReviewResult> r1 = assignmentService.getContentReviewResults(s1);
                         List<ContentReviewResult> r2 = assignmentService.getContentReviewResults(s2);
 
-                        if (CollectionUtils.isEmpty(r1)) {
+                        if (CollectionUtils.isEmpty(r1) && CollectionUtils.isEmpty(r2)) {
+                            result = 0;
+                        } else if (CollectionUtils.isEmpty(r1)) {
```

Comment:
We can't delete all of this sorting logic and just call getSortedContentReviewResults instead?

Reply:
I was initially going to do that, but that would require it to fully sort the content review items on the two submissions being compared. This just seeks the maximum score which should perform better